plone/dexterity/content.py

```diff
@@ -354,22 +354,20 @@ def _set__name__(self, value):
     __name__ = property(_get__name__, _set__name__)
 
     def UID(self):
-        """Returns the item's globally unique id."""
+        # Returns the item's globally unique id.
         return IUUID(self)
 
     @security.private
     def notifyModified(self):
-        """Update creators and modification_date.
-
-        This is called from CMFCatalogAware.reindexObject.
-        """
+        # Update creators and modification_date.
```

Comment:
ad 1: this need to go into the styleguide then
ad 2: very good reason
ad 3: another good reason
ad 4: go go go! :+1: